edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceBridge.cs

```diff
@@ -0,0 +1,320 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Cloud;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Device;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    internal class DeviceBridge
```

Comment:
>internal class DeviceBridge [](start = 4, length = 27)

let's add some boilerplate description for this class

Reply:
Will do.